src/actions/index.js

```diff
@@ -11,8 +11,16 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {
+    type: INCREMENT,
+    // payload: 1
+  }
```

Comment:
Great job making the distinction here that you can pass an payload here but it would not be necessary!